### PR TITLE
add validation that wellcomeprize can read.

### PR DESCRIPTION
**Pull request by kdaily**

### CODE REVIEW

File: workflows/scoring/validate.cwl [hunk 1]
```diff
@@ -99,7 +100,19 @@ requirements:
...
                 else:
                     return None
+            
+            def check_permissions(syn, sub):
+                invalid_reasons = []
+                project_id = sub['entityId']
+
+                try:
+                    perms = syn.getPermissions(project_id, 3380061)
+                    if "READ" not in perms:
```
Comment (philerooski) [inline workflows/scoring/validate.cwl]: DOWNLOAD

File: workflows/scoring/validate.cwl [hunk 2]
```diff
@@ -99,7 +100,19 @@ requirements:
...
+                        invalid_reasons.append("User 'wellcomeprize' cannot read the project.")
+                except synapseclient.exceptions.SynapseHTTPError as e:
+                    invalid_reasons.append("User 'wellcomeprize' cannot read the project.")
 
+                return invalid_reasons
```
Comment (philerooski) [inline workflows/scoring/validate.cwl:115]: newline

File: workflows/scoring/validate.cwl [hunk 3]
```diff
@@ -99,7 +100,19 @@ requirements:
...
+            def check_permissions(syn, sub):
+                invalid_reasons = []
+                project_id = sub['entityId']
+
+                try:
+                    perms = syn.getPermissions(project_id, 3380061)
+                    if "READ" not in perms:
+                        invalid_reasons.append("User 'wellcomeprize' cannot read the project.")
+                except synapseclient.exceptions.SynapseHTTPError as e:
+                    invalid_reasons.append("User 'wellcomeprize' cannot read the project.")
```
Comment (philerooski) [inline workflows/scoring/validate.cwl]: `"User 'wellcomeprize' does not have DOWNLOAD permissions on {}".format(project_id)`

add team ID for wellcome eval panel

File: workflows/scoring/validate.cwl [hunk 4]
```diff
@@ -113,6 +126,9 @@ requirements:
                     invalid_reasons = ["A wiki does not exist for this project. ",
                                        "Please use the template syn17024264 "
                                        "as the root wiki page."]
+
+                invalid_reasons += check_permissions(syn, sub)
```
Comment (philerooski) [inline workflows/scoring/validate.cwl:126]: Move immediately after syn.login

**Review by philerooski — CHANGES_REQUESTED** [review]

File: workflows/scoring/validate.cwl (hunk 1 above)
Comment (kdaily) [reply workflows/scoring/validate.cwl]: made it have both

**Comment by kdaily** [comment]
Take a look again @philerooski, thanks!

File: workflows/scoring/validate.cwl [hunk 5]
```diff
@@ -99,20 +100,43 @@ requirements:
...
                     return None
+            
+            def check_permissions(syn, sub):
+                invalid_reasons = []
+                project_id = sub['entityId']
+
+                try:
+                    perms = syn.getPermissions(project_id, 3385462)
+                    if "READ" not in perms and "DOWNLOAD" not in perms:
+                        invalid_reasons.append("The evaluation panel team cannot read the project.")
```
Comment (philerooski) [inline workflows/scoring/validate.cwl]: Let's include the full team name here and down below: "Wellcome Data Re-use Prize - Surveillance Evaluation Panel". That way participants can copy paste into the web UI.

**Review by philerooski — CHANGES_REQUESTED** [review]

**Review by philerooski — APPROVED** [review]

**Pull request merged**